format/testfiles/v1/test_contains.rego

```diff
@@ -0,0 +1,13 @@
+package test["contains"]
```

Comment:
Maybe change this to `package test.contains` and the formatter makes it `test["contains"]`

Reply:
Can't do that, as the parser is operating under v1 rules and would reject that package ref.

Reply:
So `package test.contains` will be a parser error in `v1` but when formatted using `--rego-v1` flag, the path will get converted to `package test["contains"]`

Reply:
correct 👍